### PR TITLE
Feature#5 login layout

### LINKED ISSUE #5: Login-Maquetación
# REQUIREMENTS
El usuario podrá iniciar sección a través de su usuario y contraseña, o a también con la cuenta de google.
Validaciones del diseño

# DESIGN
image

### PR DESCRIPTION
**Pull request by Jonathanrrs**
# Description
What did you implemented/Why did you implemented this:
 - We create the layout for the user login screen
 - We add the email and password fields
 - We add the buttons to enter by your data or through google
 - We add the design validations
 - We add the loading modal
 
 
  ## Screenshots 
image

### CODE REVIEW

**Review by LuisGomez89 — APPROVED** [review]

**Review by xManuel007 — APPROVED** [review]

File: abcorp/src/components/Login/ButtonsLogin.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  const isFormComplete = () => {
+    if (email.length > 0 && password.length > 0) {
+      setFormComplete(true);
+    } else {
+      setFormComplete(false);
+    }
+  };
+
+  useEffect(() => {
+    isFormComplete();
```
Comment (DemisRincon) [inline abcorp/src/components/Login/ButtonsLogin.js:17]: unecessary function on use effect you will get the same functionality if you set formComplete={isFormComplete()} avoiding us another state for the current state of the form

**Review by DemisRincon — COMMENTED** [review]
just a little comment , in general looks good

File: abcorp/src/components/Login/ButtonsLogin.js (hunk 1 above)
Comment (Jonathanrrs) [reply abcorp/src/components/Login/ButtonsLogin.js:17]: We are using the useEffect to verify each input when it is written, that is why we pass the onChange as a dependency, we do not check it until the button is pressed, but before in order to have the button enabled.

Comment (Jonathanrrs) [reply abcorp/src/components/Login/ButtonsLogin.js:17]: @DemisRincon We are using the useEffect to verify each input when it is written, that is why we pass the onChange as a dependency, we do not check it until the button is pressed, but before in order to have the button enabled.

**Review by DemisRincon — APPROVED** [review]

**Review by Kejoh — APPROVED** [review]
Good work

**Review by Etneciv — APPROVED** [review]
Nice components order!

**Review by osvilla27 — COMMENTED** [review]
Good Job!

**Pull request merged**